### PR TITLE
Logstash subscriber implementation

### PR DESCRIPTION
**Pull request by Spring3**
Logstash chain link for ``logtify`` logger

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    @param configs {Object} - LoggerChain configuration object
+    @param utility {Object} - Logtify common rules object
+  **/
+  constructor(configs, utility) {
+    this.settings = configs || {};
+    this.utility = utility;
+    this.winston = new winston.Logger();
+    this.winston.add(winston.transports.Logstash, {
+      port: 5000,
+      host: 'logstash.dialonce.net'
```
Comment (jkernech) [inline src/index.js]: Those values should be customisable via environment variables

**Review by jkernech — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (Spring3) [reply src/index.js]: Oh, yes, thanks. I totally forgot to move them to envs

File: README.md [hunk 2]
```diff
@@ -1,2 +1,51 @@
-# node-logtify-logstash
+# logtify-logstash
```
Comment (mrister) [inline README.md]: would be nice to add some badges to the readme. but can be another PR

File: package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "eslint": "^3.16.1",
+    "eslint-config-airbnb": "^13.0.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^2.2.3",
+    "eslint-plugin-react": "^6.8.0",
+    "istanbul": "^0.4.5",
+    "mocha": "^3.2.0",
+    "sinon": "^1.17.6"
+  },
+  "dependencies": {
```
Comment (mrister) [inline package.json]: Minor: make dependencies appear before in the package.json than devDeps.

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    this.settings = configs || {};
+    this.utility = utility;
+    if (this.settings.LOGSTASH_HOST && this.settings.LOGSTASH_PORT) {
+      this.winston = new winston.Logger();
+      this.winston.add(winston.transports.Logstash, {
+        port: parseInt(this.settings.LOGSTASH_PORT, 10),
+        host: this.settings.LOGSTASH_HOST
+      });
+    } else {
+      console.warn('Logstash logging was not initialized due to a missing token');
```
Comment (mrister) [inline src/index.js]: there is a token for logstash ?

**Review by mrister — APPROVED** [review]

File: src/index.js (hunk 4 above)
Comment (jkernech) [reply src/index.js]: Nope there is no token, the message needs to be updated

**Review by jkernech — CHANGES_REQUESTED** [review]
Agree with the small adjustments proposed by Mihovil, on the other hand the error message needs to be fixed

**Review by mrister — APPROVED** [review]

**Review by ofkindness — APPROVED** [review]

**Review by jkernech — APPROVED** [review]
@Spring3 ⚡️ 
Please go ahead and merge this one

**Pull request merged**